govuk-one-login/authentication-stubs
# Aut 3932/stub errors

## What

- Added radio buttons to ipv authorize stub page to choose desired response from ipv-reverification
- This is then passed through to ipv-token to then be passed to the ipv-reverification lambda
- If unsuccessful response was chosen, the error result is returned from ipv-reverification
- Currently this is not handled on the FE so a 'There's a problem screen' is shown - this is expected

## How to review

1. Code Review
2. Deploy to dev-new
3. Go through reset mfa journey to ipv authorize stub page
4. Choose desired responses
5. See that the appropriate responses are logged in cloudwatch
6. See that responses match those described in the [ticket](https://govukverify.atlassian.net/browse/AUT-3932)
